src/cgns_header.h

```diff
@@ -31,6 +31,7 @@ typedef char char_66[66]; /* 32 + '/' + 32 + '\0' */
 #else
 typedef char char_66[33]; /* 32 + '\0' (caller's malloc compat issues) */
 #endif
+typedef char char_md[CG_MAX_GOTO_DEPTH*33]; /* ** FAMILY TREE ** */
```

Comment:
Should be / + 32 * MAX GOTO + ending / + \0 which makes 33*MAX_GOTO + 2

Reply:
done